File: lean.lua
LEAN = {}
local lean = 'abcdefghijklmnopqrstuvwxyz'
local _lean = {"😆","🙂","🤗","😱","🤢","🙄","🗿","🥺","😒","👿","👍","🎂","💀","🤤","🤮","😷","👽","🐒","🎈","🎗","🎟","💨","💞","💜","🛑","💥"}
LEAN.run = function(lean_)
    for i = 1,#lean,1 do
        code = string.gsub(lean_,"LEAN".._lean[i],string.sub(lean,i,i))
    end
    loadstring(lean_)()
end

return LEAN

File: LEAN.py
code = ""
lean = 'abcdefghijklmnopqrstuvwxyz'
_lean = ["😆","🙂","🤗","😱","🤢","🙄","🗿","🥺","😒","👿","👍","🎂","💀","🤤","🤮","😷","👽","🐒","🎈","🎗","🎟","💨","💞","💜","🛑","💥"]
def run(lean_):
    for i in range(len(lean)):
        lean_ = lean_.replace("LEAN"+_lean[i],lean[i:i+1])
    exec(lean_)

def leanify(lean_):
    for i in range(len(lean)):
        lean_ = lean_.replace(lean[i:i+1],"LEAN"+_lean[i])
    return lean_